Added warning about correct storage device when saying safe to write (#3529)
raspberrypi/documentation: documentation/asciidoc/computers/getting-started/install.adoc

image::images/imager/are-you-sure.png[alt="Confirming a reimage of a storage device in Imager."]

> If you see an admin prompt asking for permissions to read and write to your storage medium, it's safe to proceed (provided that you have definitely selected the correct storage device earlier).

Reviewer: ```suggestion
If you see an admin prompt asking for permissions to read and write to your storage medium, grant Imager the permissions to proceed.
```

I see now. I think you're correct that "it's safe to proceed" is too strong -- and potentially misleading. But we've already dropped a lot of warnings about overwriting storage devices earlier in this passage.

Consider this rewrite instead, which satisfies your core concern (that we're telling folks an irreversible action is _safe_) without polluting the narrative with additional asides and parentheticals.

Reviewer 2: > I see now. I think you're correct that "it's safe to proceed" is too strong -- and potentially misleading. But we've already dropped a lot of warnings about overwriting storage devices earlier in this passage.
> 
> Consider this rewrite instead, which satisfies your core concern (that we're telling folks an irreversible action is _safe_) without polluting the narrative with additional asides and parentheticals.

This would satisfy my concerns around conciseness and effectiveness.

Author: This seems like better wording than my suggestion. I think someone should adopt this suggestion. Thanks for the suggestion.
